packages/ai-jsx/readme.md

```diff
@@ -1,6 +1,127 @@
-# AI.JSX
```

Comment:
Is this just a clone of the root-level README? If so, let's just use a symlink rather than copy/paste.